apache/superset
# feat: configure force_screenshot

### SUMMARY


Allow users to force a screenshot (bypass the cache) on reports that send charts.

We're not enabling this for dashboards yet, since it still needs backend support.

For alerts, this is always on for charts, and off for dashboards (for perf reasons, and also because it's not implemented).

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


![Screenshot 2021-12-12 at 12-46-09  DEV  Superset](https://user-images.githubusercontent.com/1534870/145739899-44f39c43-555b-4516-b6c3-eb9e99da3237.png)


### TESTING INSTRUCTIONS


1. Create a report, toggle the new checkbox.
2. Check that `force_screenshot` is updated accordingly in the DB, or run celery and check that screenshot URLs have `force=true` or `force=false`.

### ADDITIONAL INFORMATION
